### DISCUSSION TITLE
Make text italic in TextColumn

### DISCUSSION THREAD

**Discussion by Wulfheart** [filamentphp/filament#5436] [Help] [answered]
Hello,

if an Entry in a TextColumn is null I would like to format the text italic. I think this should work with a callback function. However, I am unable to find instructions on how to make text italic in the docs. Is there any hidden method for it, is it just not supported or am I just unable to find it?

Thank you in advance.

Best
Alex

**Comment by danharrin** [comment, marked as answer]
I don't think theres a dedicated method for it, so try `->extraAttributes(['class' => 'italic'])`
